lib/reporters/html.js

```diff
@@ -172,7 +277,7 @@ internals.Reporter.prototype.end = function (notebook) {
 internals.findLint = function (lint, file) {
 
     for (let i = 0; i < lint.length; ++i) {
-        if (lint[i].filename.slice(-file.filename.length) === file.filename && lint[i].errors.length) {
+        if (lint[i].filename.replace(/\\/g, '/').slice(-file.filename.length) === file.filename && lint[i].errors.length) {
```

Comment:
`lint.filename` is platform dependant, which is ok (notably when we fix lint issues), but file.filename always use `/`, so we need to to the replacement.